### ISSUE TITLE
Field mysteriously not showing in reviews

### ISSUE THREAD

**Issue by rfonseca** [kohler/hotcrp#97]
I created a review form for a class I am teaching, by editing an existing review form.

One of the text fields, 'Questions(s)', does not appear when viewing the reviews normally, although it does appear when viewing the reviews in plain text. Some reviewers do leave this field blank, but others don't.

I attached the review form as it is in the database for reference. I tried changing the visibility of the field, to no avail.

This is the field description:
~~~~
"commentsToAddress": {
    "name": "Question(s)",
    "description": "Write one or two questions that this paper raises for you. We will try to discuss this in class.",
    "position": 6,
    "visibility": "au"
  },
~~~~

Am I missing anything?

Attached also is an image of the review form editing screen. I'm using HotCRP v2.100 [6c8f753...]

[reviewform.txt](https://github.com/kohler/hotcrp/files/777320/reviewform.txt)

![image](https://cloud.githubusercontent.com/assets/163015/22978337/5a34eeb4-f360-11e6-8328-583d82a4d70c.png)

**Comment by kohler** [comment]
Can you create me an account so I can see this?

You also could try updating to HEAD. I don't remember any problem with this symptom, however.

**Comment by rfonseca** [comment]
Just created you a sysadmin account.

More investigation on the symptoms:
- If I name the field 'Question(s)', Question(, Question), or (), it does not show up.
- The position of the field in the form does not change the behavior
- I created another text field, named it Question(s), and it also didn't show up.
- I changed the name of the field Evidence to Evidenc(e)... and it showed up ! :(

The current work around is that I changed the name of the field to 'One or more questions', and it shows up. Still don't understand.

Thanks!
Rodrigo
